runtime/isp_run_app.py

```diff
@@ -39,6 +39,13 @@ def main():
     parser.add_argument("-p", "--policy", type=str, default="none", help='''
     Name of the installed policy to run or directory containing policy. Default is none
     ''')
+    parser.add_argument("-P", "--policy-full", type=str, help='''
+    Full name of the installed policy to run or directory containing policy. Default
+    is to defer to normal -p behavior. If this is specified, -p is redundant/ignored.
+    ''')
+    parser.add_argument("-k", "--kernels", type=str, help='''
```

Comment:
Question: Do we think this makes more sense as a "look here" parameter or a "look here first, then go to ISP_PREFIX" type parameter?

Reply:
See above. If the option passed to `-p` is a directory, then `isp_run_app` uses the directory instead of looking in `ISP_PREFIX`

Reply:
This instruction assumes that the kernels are found in `ISP_PREFIX` — https://github.com/draperlaboratory/hope-tools/blob/master/runtime/isp_run_app.py#L123

Again, I want application-specific PEX kernels to live in my project’s directory rather than my `ISP_PREFIX`